Ubuntu build not working

With the latest Conan changes, running bootstrap-orbit.sh fails at the cmake "generate" step.

I get: 

```
-- Configuring done
CMake Error at OrbitGl/CMakeLists.txt:4 (add_library):
  Target "OrbitGl" links to target "OpenGL::OpenGL" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
```

I get the same when running "build.sh clang9_release" for example.

--- a/OrbitGl/CMakeLists.txt
+++ b/OrbitGl/CMakeLists.txt
@@ -118,7 +118,7 @@ else()
   set_target_properties(OrbitGl PROPERTIES EXCLUDE_FROM_ALL ON)
 endif()
 
-if(TARGET OpenGL::GLX AND WITH_GUI)
+if(TARGET OpenGL::GLX AND TARGET OpenGL::OpenGL AND WITH_GUI)
   target_link_libraries(OrbitGl PUBLIC OpenGL::GLX)
 endif()
 

